coralproject/talk
# shortcuts list should be accessible from the cog

## What does this PR do?

I forgot to put a link to the shortcut modal in the cog like @IAmSamHankins had asked. Now it works!

## How do I test this PR?

- go to the admin
- click the cog in the upper right
- there should be a menu option for Shortcuts
- it should display the shortcuts modal
